localstripe/resources.py

```diff
@@ -753,6 +753,9 @@ def __init__(self, name=None, description=None, email=None,
                 assert type(data['value']) is str and len(data['value']) > 10
             if payment_method is not None:
                 assert type(payment_method) is str
+            if balance is not None:
+                balance = -try_convert_to_int(balance)
```

Comment:
When looking at the surrounding lines, and also at the rest of the file, `try_convert_to_int()` is usually not used like this, and the default value `0` neither.

Can you change it to:
\- set the default value `balance=0` in the function signature,
\- set `balance = try_convert_to_int(balance)` before assertions,
\- inside check its type, without `if balance is not None:`,
\- set the negative inside `self.account_balance = -balance`?

Reply:
As you wish, as long as the linter agrees :wink: 

(I see we already use both in our code base.)